frontend/src/views/SecretOverviewPage/components/CreateSecretForm/CreateSecretForm.tsx

```diff
@@ -133,6 +135,21 @@ export const CreateSecretForm = ({
       });
     }
   };
+
+  const handlePaste = (e: ClipboardEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const pastedContent = e.clipboardData.getData('text');
+    const splitIndex = pastedContent.indexOf("=");
+
+    if (splitIndex === -1) return
+
+    const key = pastedContent.slice(0, splitIndex)
+    const value = pastedContent.slice(splitIndex + 1);
+
+    setValue('key', key);
+    setValue('value', value);
```

Comment:
Please use double quotes ("), instead of single quotes (').

Single quotes will result in a linting error.